server/db/schema.js

```diff
@@ -110,6 +112,34 @@ const schema = {
       )`,
     },
   ],
+  phaseTables: [
+    {
+      definition: `CREATE TABLE IF NOT EXISTS ${collections.GLOBAL_PHASE} (
+        id serial primary key,
+        name varchar(255) not null,
+        start_date date not null,
+        end_date date not null,
+        created_by varchar(255) not null,
+        updated_by varchar(255) not null,
+        created_at timestamp with time zone DEFAULT now(),
+        updated_at timestamp with time zone DEFAULT now()
+      )`,
+    },
+    {
+      definition: `CREATE TABLE IF NOT EXISTS ${collections.SITE_PHASE_ALLOCATION} (
+        id serial primary key,
+        phase_id integer references ${collections.GLOBAL_PHASE},
+        site_id integer references ${collections.EMPLOYER_SITES},
+        allocation integer not null,
+        start_date date,
+        end_date date,
+        created_by varchar(255) not null,
+        updated_by varchar(255) not null,
+        created_at timestamp with time zone DEFAULT now(),
+        updated_at timestamp with time zone DEFAULT now()
```

Comment:
Do we want to make these not null? (I see how it matters less with `DEFAULT` but would prevent accidental data deletion

Reply:
definitely.